tools/integration_tests/run_e2e_tests.sh

```diff
@@ -63,7 +63,8 @@ TEST_DIR_PARALLEL=(
   "interrupt"
   "operations"
   "log_content"
-  "kernel-list-cache"
+  "kernel_list_cache"
+  "concurrent_operations"
```

Comment:
nit: Is "concurrent_operations" expected in the diff? Should it be in a different PR?

Reply:
Yeah, should have done this in a separate PR. I'll take care next time.